src/test/java/wooteco/subway/dao/JdbcLineDaoTest.java

```diff
@@ -0,0 +1,123 @@
+package wooteco.subway.dao;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.JdbcTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import wooteco.subway.domain.Line;
+
+@JdbcTest
+public class JdbcLineDaoTest {
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+    private LineDao lineDao;
+
+    @BeforeEach
+    void setUp() {
+        lineDao = new JdbcLineDao(jdbcTemplate);
+    }
+
+    @Test
+    @DisplayName("Line 을 저장한다.")
+    void save() {
+        //given
+        Line line = new Line("7호털", "khaki");
+
+        //when
+        Line actual = lineDao.save(line);
+
+        //then
+        assertThat(isSameNameAndColor(actual, line)).isTrue();
```

Comment:
```suggestion
        assertThat(actual).usingRecursiveComparison()
                .ignoringExpectedNullFields()
                .isEqualTo(line);
```

이런 방법, 또는

```suggestion
        assertThat(actual).usingRecursiveComparison()
                .ignoringFields("id")
                .isEqualTo(line);
```

이런 방법도 있어요. 참고만 해 주세요 😎

Reply:
어떻게 `equals, hashcode`를 클래스에 재 정의하지 않고 속성 값들을 비교할지 고민을 많이 했습니다🤔. 그 결과 테스트에 만든 메소드였는데 마음에 들지는 않았습니다. `코니`가 알려주신 방법이 훨씬 깔끔하고 확장에도 대응 가능하다고 생각하여 도입해보았습니다!!! 감사합니다👍👍